### PR TITLE
rule:PrintNightmare - Registry events

### PR DESCRIPTION
**Pull request by kidrek**

### CODE REVIEW

File: rules/windows/registry_event/win_registry_printnightmare.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    TargetObject|contains:
+      - '\Control\Print\Environments\Windows x64\Drivers\Version-3\'
+  selection_end:
+    TargetObject|endswith:
+      - '\Configuration File'
+      - '\Data File'
+  selection_dll:
+    Details|endswith:
+      - '.dll'
+  condition: all of them
```
Comment (frack113) [inline rules/windows/registry_event/win_registry_printnightmare.yml:35]: Why not
```yaml
  selection_event:
    Image|endswith: '\spoolsv.exe'
    EventType: 'SetValue'
    TargetObject|contains: '\Control\Print\Environments\Windows x64\Drivers\Version-3\'
    Details|endswith: '.dll'
  selection_end:
    TargetObject|endswith:
      - '\Configuration File'
      - '\Data File'
```

**Comment by Neo23x0** [comment]
Why wouldn't this trigger on just any printer driver installation?

File: rules/windows/registry_event/win_registry_printnightmare.yml (hunk 1 above)
Comment (kidrek) [reply rules/windows/registry_event/win_registry_printnightmare.yml]: Perfect @frack113 !! thx

**Comment by kidrek** [comment]
> Why wouldn't this trigger on just any printer driver installation?

I didn't identify any false positives during my tests.
I installed different types of printers and i only found ".gpd" files usage for the drivers configuration.
But i'm still testing to identify possible false positives.

**Comment by frack113** [comment]
Do you have updates on the tests you have done ?

**Comment by kidrek** [comment]
> Do you have updates on the tests you have done ?

I had false positives during my last test. 
It would be better not to put it in production. I close this pull request for this moment.
I'll still working on it.

**Pull request closed without merging**